config/_default/params.yaml

```diff
@@ -19,5 +22,7 @@ showRecentPosts: true
 recentNews: 5 # number of recent news to show in home page
 showRecentNews: true
 
-social_navbar: true # show social icons on navbar
-social_index: true # show social icons on index page
+socialNavbar: true # show social icons on navbar
+socialIndex: true # show social icons on index page
+
+colorTheme: blue # choose a theme from `data/themes` folder
```

Comment:
I think it is better to have a default value so that users understand how to change the color theme. 

Also, I deleted `default.yaml`. Default doesn't mean anything. I think we should name all themes according to some color.

Reply:
The only reason I was using default is because the normal themes are failing the lighthouse accessibility tests, see https://app.netlify.com/sites/academic-folio/deploys/63d3ab5551c1480008672f95

Reply:
It is great that you mentioned this. Let's merge this request however you like and I will deal with this accessibility score later.